packages/translations/locales/de/admin.json

```
@@ -1540,5 +1541,135 @@
+      "pause": {
+        "title": "Abonnement pausieren",
+        "description": "Dieses Abonnement pausieren. Der Abonnent verliert den Zugang bis zur Fortsetzung."
+      },
+      "cancel": {
+        "title": "Abonnement kündigen",
+        "description": "Dieses Abonnement kündigen. Diese Aktion kann durch Verlängerung rückgängig gemacht werden."
+      },
+      "renew": {
+        "title": "Abonnement verlängern",
+        "description": "Dieses Abonnement für einen weiteren Abrechnungszeitraum verlängern."
+      },
```

Biome (2.1.2)

[error] The key cancel was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)